### PR TITLE
tests: add regression tests for nil pointer on invalid kubeconfig

### PR DESCRIPTION
**Pull request by 89luca89**
/kind test

Add regression test for nil pointer on invalid kubeconfig

### CODE REVIEW

**Review by FabianKramm — CHANGES_REQUESTED** [review]

File: e2e/tests/pipelines/pipelines.go [hunk 1]
```diff
@@ -418,3 +418,30 @@ var _ = DevSpaceDescribe("pipelines", func() {
 		framework.ExpectNoError(err)
 	})
 })
+
+var _ = DevSpaceDescribe("pipelines_without_kubeconfig", func() {
```
[the remark this answers is not in the document]
Comment (89luca89) [reply e2e/tests/pipelines/pipelines.go]: Done :+1:
